### PR TITLE
[tf.keras] Remove relu6 and unused imports

### PR DESCRIPTION
**Pull request by d02k01**

### CODE REVIEW

**Review by qlzh727 — DISMISSED** [review]

**Comment by qlzh727** [comment]
Thanks for the cleanup.

**Comment by qlzh727** [comment]
Can u fix the failed test cases?

**Comment by mktozk** [comment]
Fixed. Please review it.

File: tensorflow/python/estimator/keras_test.py [hunk 1]
```diff
@@ -477,14 +477,8 @@ def test_custom_objects(self):
     keras_mobile = mobilenet.MobileNet(weights=None)
     keras_mobile.compile(loss='categorical_crossentropy', optimizer='adam')
     custom_objects = {
-        'relu6': mobilenet.relu6,
         'DepthwiseConv2D': mobilenet.DepthwiseConv2D
     }
-    with self.assertRaisesRegexp(ValueError, 'relu6'):
```
Comment (qlzh727) [inline tensorflow/python/estimator/keras_test.py]: Instead of removing this, should it just expect "DepthwiseConv2D" in the exception?

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: tensorflow/python/estimator/keras_test.py (hunk 1 above)
Comment (mktozk) [reply tensorflow/python/estimator/keras_test.py]: Thanks! I fixed it.

Comment (mktozk) [reply tensorflow/python/estimator/keras_test.py]: `DepthwiseConv2D` is also no longer custom objects. So mobilenet is not suitable for `test_custom_objects`. I'll fix it to define simple model in `keras_test.py`.

Comment (qlzh727) [reply tensorflow/python/estimator/keras_test.py]: Ack. Please fix the test case.

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**